clients/src/main/java/org/apache/kafka/clients/producer/KafkaProducer.java

```diff
@@ -224,7 +224,7 @@
  */
 public class KafkaProducer<K, V> implements Producer<K, V> {
 
-    private static final Logger log = LoggerFactory.getLogger(KafkaProducer.class);
+    private static Logger log;
```

Comment:
Unlike `KafkaConsumer`, `KafkaProducer` has some private static methods where use `log` such as `configureRetries` or `configureInflightRequests`, so should we modify these methods also?